src/main/webapp/utilities/SessionManager.js

```diff
@@ -0,0 +1,30 @@
+export default class SessionManager {
+    constructor(localStorageObject = window.localStorage, localStorageKey) {
+        this.localStorage = localStorageObject;
+
+        this.localStorageKey = localStorageKey;
+    }
+
+    /**
+     * Gets JSON-parsed value from LocalStorage
+     * @returns {Any} the value on that field
+     */
+    getValueInLocalStorageParsed() {
+        return JSON.parse(this.localStorage.getItem(this.localStorageKey));
+    }
+
+    /**
+     * Sets value to LocalStorage
```

Comment:
Add to the docstring that this will overwrite the existing value if called again

Reply:
Done! :)